notebooks/MIRI/JWPipeNB-MIRI-MRS.ipynb

```diff
@@ -23,8 +23,8 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.    # Turn on multi-core processing for jump step (off by default).  Choose what fraction of cores to use (quarter, half, or all)
</pre><p>Do you still mean this to say turn on multi-core processing? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwst-pipeline-notebooks/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
Yes, it's fine for the jump step, just not for ramp fitting.